.gitignore

```diff
@@ -3,3 +3,4 @@ phpunit.xml
 composer.lock
 composer.phar
 vendor/*
+.idea/
```

Comment:
Should be reverted (you can ignore environment specific files and directories locally).